lib/wasi-web/src/pool.rs

```diff
@@ -79,7 +80,7 @@ struct WasmRunCommand {
     snapshot: Option<InstanceSnapshot>,
     trigger: Option<WasmRunTrigger>,
     update_layout: bool,
-    result: Result<(), Errno>,
+    result: Option<Result<Bytes, ExitCode>>,
```

Comment:
Can we have a comment here explaining what this `result` is?

Pretty confusing if you don't know the context. (why `Option`, why `Bytes`, ...)

Reply:
Comments added